packages/pluggableWidgets/background-gradient-native/src/BackgroundGradient.tsx

```diff
@@ -13,7 +13,37 @@ export function BackgroundGradient({ name, colorList, content, onClick, style }:
         Alert.alert("", "The widget 'Background gradient' requires an updated 'Make It Native 9' application");
     }
 
+    const opacityValidation = (opacity: number | undefined): number => {
```

Comment:
this function doesn't have to be inner function. I would move it out of the component for testability and readability.

Reply:
done